docs/security.md

```diff
@@ -122,7 +122,7 @@ vectors including resource exhaustion.
 
 Component developers MUST default to encrypted connections (via the `insecure:
 false` configuration setting) and SHOULD leverage
-[receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/receiverhelper)
+[receiver](https://github.com/open-telemetry/opentelemetry-collector/blob/main/component/receiver.go)
```

Comment:
config[grpc|http] packages I believe are relevant here, not the helpers to build the components.

Reply:
Updated to link to configgrpc and confighttp packages.